front50-web/src/main/groovy/com/netflix/spinnaker/front50/controllers/v2/ApplicationsController.groovy

```diff
@@ -134,6 +134,8 @@ public class ApplicationsController {
       def perm = applicationPermissionDAO?.findById(app.name)
       if (perm?.permissions?.isRestricted()) {
         app.details().put("permissions", perm.permissions)
+      } else {
+        application.details().remove("permissions")
```

Comment:
does this work ... `application` vs `app`?

Reply:
nope 😳 .... fixed in #514 thank you!